isis/src/control/apps/jigsaw/jigsaw.xml

```diff
@@ -24,53 +25,371 @@
+      become correlated and there will be multiple minimums for the observation equations.
+      We recommend holding the instrument position fixed becaue a-priori data for it generally is more
+      accurate than the instrument pointing.
+    </p>
+
+    <h4>Sensor Types</h4>
+    <h5>Framing</h5>
+    <p>
+      Framing sensors are the most basic type or sensor to use with <b>jigsaw</b>. The entire image is exposed at the same
+      time, so there is only a single position and pointing value for each image. With these datasets
+      you will want to use the either NONE or POSITIONS for the <b>SPSOLVE</b> parameter and ANGLES
+      for the <b>CAMSOLVE</b> parameter.
+    </p>
+    <h5>Pushbroom</h5>
+    <p>
+      Pushbroom sensors require additional consideration because the image is exposed over a period of
+      time and therefore, position and pointing are time dependent. The <b>jigsaw</b> application handles this
+      by modeling the position and pointing as polynomial functions of time. Polynomial functions
+      are fit to the a-priori position and pointing values on each image. Then, each iteration updates
+      the polynomial coefficients. Setting <b>CAMSOLVE</b> to velocities will use first
+      degree, linear, polynomials and setting <b>CAMSOLVE</b> to accelerations will use
+      second degree, quadratic, polynomials for the pointing angles. For most pushbroom observations,
+      we recommend using second degree polynomials, solving for accelerations.
+    </p>
+    <p>
+      There are also a variety of parameters that allow for more precise control over the polynomials used
+      and the polynomial fitting process. The <b>OVERHERMITE</b> parameter changes the initial polynomial
+      setup so that it does not replace the a-priori position values. Instead, the polynomial starts out as a
+      zero polynomial and is added to the original position values. This effectively adds the noise
+      from the a-priori position values to the constant term of the polynomial. The <b>OVEREXISTING</b>
+      parameter does the same thing for pointing values. These two options allow you to preserve complex
+      variation like jitter. We recommend enabling these options for jittery images and very long
+      exposure pushbroom images.
+    </p>
+    <p>
+      The <b>SPSOLVE</b> and <b>CAMSOLVE</b> parameters only allow up to a second degree
+      polynomial. To use a higher degree polynomial for pointing values, set <b>CAMSOLVE</b> to ALL and then set
+      the <b>CKDEGREE</b> parameter to the degree of the polynomial you want to fit over the original
+      pointing values and set the <b>CKSOLVEDEGREE</b> parameter to the degree of the polynomial you want
+      to solve for. Setting the <b>SPSOLVE</b>, <b>SPKDEGREE</b>, and <b>SPKSOLVEDEGREE</b> parameters
+      in a similar way adjusts the polynomial for postion values in the same way.
```

Comment:
'position'

Reply:
This is supposed to be position. The previous section talks about pointing and this one talks about position.